library/classes/Totp.class.php

```diff
@@ -14,6 +14,7 @@
 
 use RobThree\Auth\Providers\Qr\BaconQrCodeProvider;
 use RobThree\Auth\TwoFactorAuth;
+use RobThree\Auth\Algorithm;
```

Comment:
nice fix @DrBassman ! 

1 minor thing that is mostly followed is the alpha ordering of the use statements

```
use RobThree\Auth\Algorithm;
use RobThree\Auth\Providers\Qr\BaconQrCodeProvider;
use RobThree\Auth\TwoFactorAuth;

Reply:
I'm still new to this & appreciate help "getting up to speed"... Do I update my changes so the use statements are in the preferred order?

Also, each time I do this should I delete my repository, and start with a newly forked version next time.  I'm a little confused how the message here says I have 6 commits to merge, when I only changed the one file?

Thank you for your help.

Reply:
sure, update the use statements if you'd like in the next commit on this branch

no need to delete anything :scream: 

next time start a new branch after you update the `master` branch

anyhow, when we go to bring in your PR all of the commits will get squashed into 1 commit so it isn't really an issue anyways :)